lib/classes/config.js

```diff
@@ -12,7 +11,7 @@ class Config {
   }
 
   update(config) {
-    return _.merge(this, config);
+    return Object.assign(this, config);
```

Comment:
Did you have a chance to trace if that would be a safe refactor here? Merge does deep copy and assign does only a shallow copy, I think such generic function as this one might be potentially dangerous to rely on just shallow copy